backend/api/v1/issue_service.go

```diff
@@ -282,6 +282,12 @@ func (s *IssueService) getIssueFind(ctx context.Context, permissionFilter *store
 				issueLabel := label
 				issueFind.LabelList = append(issueFind.LabelList, issueLabel)
 			}
+		case "no_pipeline":
```

Comment:
Use "pipeline" instead of "no_pipeline" for positive sounding.

https://testing.googleblog.com/2023/10/improve-readability-with-positive.html

Reply:
How about `has_pipeline`?